### ISSUE TITLE
Unsupported type error for `[u64 ; 4usize]`

### ISSUE THREAD

**Issue by thomaseizinger** [google/autocxx#361]
## Expected Behavior

No panics when running autocxx.

## Actual Behavior

Autocxx panics with the following error:

```
ParseError(AutocxxCodegenError(Conversion(UnsupportedType("[u64 ; 4usize]"))))
```

## Steps to Reproduce the Problem

1. Clone https://github.com/thomaseizinger/monero-wallet-sys
1. Checkout commit `1f3e4eab`
1. Run `cargo build --release` (the debug build actually panics for a different reason)

## Specifications

- Version: 0.5.4
- Platform: Linux / Manjaro

**Comment by adetaylor** [comment]
Thanks! I had a quick go at reproducing this but even after fetching all the submodules I run into difficulties with a lack of boost.

Would you mind coming up with a single preprocessed file per the [instructions here](https://github.com/google/autocxx#reporting-bugs)? A creduced file would be even better, but I'm quite happy to run the reduction myself if you can give me a preprocessed file.

I'm pretty sure this will turn out to be a duplicate of #266. However, I was already thinking of doing #364 and I'll therefore probably do that sooner.

**Comment by adetaylor** [comment]
In fact I'm so confident that this is #266 I'll just close it. Feel free to reopen if you have a preprocessed file and I'll check.

**Comment by thomaseizinger** [comment]
Hate to crush your confidence on this but it looks like it is unrelated to C++ arrays :(

I opened a PR with an integration test here: https://github.com/google/autocxx/pull/370

**Comment by adetaylor** [comment]
Aiming to fix in #373 which builds on #370.
